src/ec/packages/ec-component-timeline/timeline.html.twig

```diff
@@ -0,0 +1,106 @@
+{% spaceless %}
+
+{#
+  Parameters:
```

Comment:
the parameters described here don't match the actual parameters of the templates

Reply:
Fixed